.gitlab-ci.yml

```diff
@@ -34,3 +35,18 @@ build:make:
     - apk add --update git make gcc pkgconfig libc-dev glib-dev
   script:
     - make build
+
+trigger:mender-dist-packages:
+  image: alpine
+  stage: trigger
+  before_script:
+    - apk add --no-cache curl git
```

Comment:
Why do you need git?

Reply:
I don't need it, that was a left over from the `mender` pipeline I copied from. Thanks!